Dockerfile

```
@@ -1,5 +1,4 @@
-# Use the official Ruby 3.2.0 Alpine image as the base image
-FROM ruby:3.2.0-alpine
+FROM ruby:3.3.5-alpine
```

To float on latest 3.3.x:
```suggestion
FROM ruby:3.3-alpine
```